codecs/oxipng/build.sh

```diff
@@ -4,8 +4,12 @@ set -e
 
 rm -rf pkg,{-parallel}
 export CFLAGS="${CFLAGS} -DUNALIGNED_ACCESS_IS_FAST=1"
-wasm-pack build -t web
-RUSTFLAGS='-C target-feature=+atomics,+bulk-memory' wasm-pack build -t web -d pkg-parallel -- -Z build-std=panic_abort,std --features=parallel
+wasm-pack build -t web -- --locked
```

Comment:
What does `--locked` do?

Reply:
We already use it in default rust.Dockerfile command, just noticed we missed it here. It tells Cargo to use Cargo.lock and not check for any versions compatible with Cargo.toml range, improving reliability of builds.

Reply:
ah of course